wls-gui-wahllokalsystem/src/composables/tasks/taskFactories/wahlvorstandTaskFactory.ts

```diff
@@ -0,0 +1,26 @@
+import type { TaskFactoryContext } from "@/composables/tasks/TaskFactoryContext.ts";
+import type { TaskFactoryInterface } from "@/composables/tasks/TaskFactoryInterface.ts";
+import type { Task } from "@/types/tasks/Task.ts";
+
+import { useWahlvorstandStore } from "@/stores/wahlvorstandStore.ts";
+
+export function useWahlvorstandTaskFactory(): TaskFactoryInterface {
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  function createTasks(taskFactoryData: TaskFactoryContext): Task[] {
```

Comment:
Wusste auch nicht dass das kompatibel ist

```suggestion
  function createTasks(): Task[] {
```